src/components/OrderableList/OrderableList.stories.tsx

```diff
@@ -40,6 +41,11 @@ export default {
             options: ['left', 'right', 'none'],
             control: { type: 'inline-radio' },
         },
+        disableDragDelay: {
+            table: { category: 'Item Options' },
+            defaultValue: false,
+            control: { type: 'boolean' },
+        },
```

Comment:
```suggestion
        dragDelay: {
            table: { category: 'Item Options' },
            defaultValue: 150,
            control: { type: 'number' },
        },
```

Reply:
Hey @SamuelAlev thanks for the feedback.  @anxobotana and I spoke about a `default` time as well, but he felt it would not be applicable.  @anxobotana change your mind?

Reply:
@anxobotana felt it was excessive as no one would need a delay beyond `150ms` (https://frontify.slack.com/archives/C02PSJTPA3D/p1697015184159539?thread_ts=1696940357.966949&cid=C02PSJTPA3D)

Reply:
Could be excessive but still better than `disable[...]`

Reply:
Me being the most junior member I will defer this decision to you @SamuelAlev

Reply:
General rule with props is not to have a mix of positive and negative prop but rather one, seems like the "positive" prop approach is used heavily, so IMO go with `delay={0}` or `enableDelay={false}`

Reply:
Thx @SamuelAlev I will make the update.